intranet/templates/events/roster.html

```diff
@@ -51,7 +52,23 @@ <h2>Public Roster</h2>
 
             {% if num_hidden_members > 0 %}
                 <br>
-                <p>{{ num_hidden_members }} {% if num_hidden_members == 1 %}user {% if not request.user.can_view_eighth %}(you) {% endif %}has{% else %}users {% if not request.user.can_view_eighth %}(including you) {% endif %}have{% endif %} not given permission to appear on this list.</p>
```

Comment:
Since this will all appear on one line on the page, it's best to leave it on one line now.